### DISCUSSION TITLE
Does anyone know if this type of rendering in RDKIT ver. 2020.03.3.0 is normal

### DISCUSSION THREAD

**Discussion by Lhx0108** [rdkit/rdkit#4391] [General] [answered]
Please see the screenshot below,
![image](https://user-images.githubusercontent.com/16868854/128190250-a232ca0f-6175-4ef7-8aab-6b220e98e40c.png)
I am wondering if the carbons at the top of the structure is normal in RDKIT. The SMILES is CCC(CC(O)C(C)C1OC(=O)/C(C)=C\C=C\C(CC)C(O)/C(C)=C/C=C/C=C/C=C/C=C/C=C/C=C/CC2CC(O)CC(O)(CC(O)CC(O)CC(O)CC(O)CC(O)CC(O)CC(O)C(CC)C(O)C(C)C(O)C1C)O2)OC1CC(O)C(N(C)C)C(C)O1

Thanks

**Comment by ptosco** [comment, marked as answer]
RDKit will depict an explicit C atom when the are neither multiple bonds nor substituents that help showing the presence of a carbon atom, and the two contiguous single bonds are almost collinear, making it very difficult to figure out that there is indeed a carbon atom in between the two bonds.
If you switch to the CoordGen depiction you will get a nicer layout that does not require those explicit carbons as it has 120 degree angles between sp3 carbons:
```
from rdkit.Chem.Draw import rdDepictor
rdDepictor.SetPreferCoordGen(True)
```
![image](https://user-images.githubusercontent.com/5244385/128192668-a8b7568f-081e-436e-bd07-e5d4b9d99629.png)

**Reply by Lhx0108** [reply]
Thanks a lot! It looks awesome!
